master/static/connect29.js

```diff
@@ -0,0 +1,8 @@
+$(document).ready(() => {
+  console.log("Hello from the profile page");
+    $('#image_table_row').on('click', () => {
+      console.log("Hello from the image table row");
```

Comment:
1. How come we have 3 different connect##.js? do they have different purposes? If so, they should be named in such a way that it describes what they do.
2. What's the meaning of the log messages? I imagine you added them when you were testing out the functionality, but now they no longer look like they're needed and can be removed.